lib/formatters/tapFormatter.mjs

```diff
@@ -57,3 +59,29 @@ export default function tapFormatter(results) {
 
 	return lines.join('\n');
 }
+
+/**
+ * Apply TAP escaping to the source that is part of the test point description
+ * @see {@link https://testanything.org/tap-version-14-specification.html#escaping TAP 14 specification}
+ * @param {string} [source]
+ */
+function getEscapedSource(source) {
+	if (!source) return source;
+
+	let output = '';
+
+	for (const char of source) {
+		switch (char.codePointAt(0)) {
+			case 0x005c: // `\`
+			case 0x0023: // `#`
```

Comment:
note

This function assumes that PostCSS passes us an exploitable URI when the source isn't an id.
i.e. when the hash is not the delimiter of the fragment identifier it is properly encoded to %23